### PR TITLE
Add policies section and begin with a commit rights policy

### PR DESCRIPTION
**Pull request by eteq**
This adds a new directory called "policies", to be a place to store current policies for the Astropy Projects.

Policies in this sense mean what we do *now*, so the idea is that this space is used for "living" policies.  Right now I'm thinking of this as a space to record the existing policies (many of which are currently un-written), but can also be used for development of new or currently not clear policies.

As a "test" of this, I've also written up what I see as the current commit-writes policy (which I am informally the holder of).  Note that this document indirectly depends on #93, since this references that document, but it's probably not critical to think of it as a blocker.

### CODE REVIEW

**Comment by astrofrog** [comment]
Looks good to me! I wonder if we want to say something in the README about how this interacts with APEs?

File: policies/coordinated-write-permissions.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Write Permissions for Coordinated Package Repositories
+
+This policy describes the rules and expectations for receiving write permissions for Coordinated Packages (including the Astropy Core Package).
+
+Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be grant for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), be such permissions must be done temporarily.
```
Comment (hamogu) [inline policies/coordinated-write-permissions.md:5]: typo: be such permission - > but

Comment (nstarman) [inline policies/coordinated-write-permissions.md:5]:
```suggestion
Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be granted for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), be such permissions must be done temporarily.
```

Comment (eteq) [reply policies/coordinated-write-permissions.md:5]:
```suggestion
Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be grant for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), but such permissions must be done temporarily.
```

File: policies/coordinated-write-permissions.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Write Permissions for Coordinated Package Repositories
+
+This policy describes the rules and expectations for receiving write permissions for Coordinated Packages (including the Astropy Core Package).
+
+Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be granted for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), be such permissions must be done temporarily.
```
Comment (eteq) [inline policies/coordinated-write-permissions.md:5]:
```suggestion
Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be granted for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), but such permissions must be done temporarily.
```

File: policies/coordinated-write-permissions.md (hunk 1 above)
Comment (eteq) [reply policies/coordinated-write-permissions.md]: this conflicted with @nstarman's change, but I updated it in https://github.com/astropy/astropy-project/pull/99/commits/950b016fe55775b4f6fd1da6d818167df1cf2ab1

File: policies/coordinated-write-permissions.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Write Permissions for Coordinated Package Repositories
+
+This policy describes the rules and expectations for receiving write permissions for Coordinated Packages (including the Astropy Core Package).
+
+Github write privileges are granted when someone is given a relevant named role. In most cases this is package (or sub-package) maintainer, although other roles (for example, the astropy.org web page maintainer) may also come with write permissions for a specific named repository.  Occasional *temporary* write (or higher) permissions may be granted for specific tasks (for example, a maintainer may be given admin rights to a repo to configure CI for the first time or similar), but such permissions must be done temporarily.
+
+Normal write permissions should be given by assigning sometime to the Github Team that corresponds to that role (For Example, "Astropy Core Maintainers" or "Astropy web site maintainers" for a core maintainer or the astropy.org team, respectively), a duty primarily performed by the Coordination Committee.  Temporary permissions should instead use the "collaborator" feature on Github to make it clear that these permissions are temporary in nature.
```
Comment (hamogu) [inline policies/coordinated-write-permissions.md:7]:
```suggestion
Normal write permissions should be given by assigning sometime to the Github Team that corresponds to that role (For example, "Astropy Core Maintainers" or "Astropy web site maintainers" for a core maintainer or the astropy.org team, respectively), a duty primarily performed by the Coordination Committee.  Temporary permissions should instead use the "collaborator" feature on Github to make it clear that these permissions are temporary in nature.
```

**Pull request merged**